matplotcheck/base.py

```diff
@@ -376,89 +377,14 @@ def assert_legend_labels(self, labels_exp):
         legend_texts = [
             t.get_text().lower() for leg in legends for t in leg.get_texts()
         ]
+        labels_exp = [l.lower() for l in labels_exp]
         assert len(legend_texts) == len(
             labels_exp
         ), "Legend does not contain expected number of entries"
         assert set(legend_texts) == set(
             labels_exp
         ), "Legend does not have expected labels"
 
-    def which_label(self, label, all_label_options):
```

Comment:
@kysolvik were these methods removed intentionally?

Reply:
i don't think we want to remove these. it looks you this section has been deleted. unless there is good justification for this can you please replace this code?

Reply:
see note above -- duplicate method.